packages/react-components/src/components/MIMap/MapboxMap/MapboxMap.jsx

```diff
@@ -36,8 +37,9 @@ MapboxMap.propTypes = {
  * @param {Object} [props.mapsIndoorsInstance] - Instance of MapsIndoors class: https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.MapsIndoors.html
  * @param {Object} [props.viewModeSwitchVisible] - Set to true to show the view mode switch.
  * @param {Object} [props.mapOptions] - Options for instantiating and styling the map as well as UI elements.
+ * @param {Object} [props.appConfig] - Object that contains app settings.
```

Comment:
```suggestion
 * @param {Object} [props.appConfig] - Object that contains app config.
```

(it's a MapsIndoors term 😄 )